src/index.html

```diff
@@ -1,13 +1,324 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBike</title>
+    <link rel="icon"
+      type="image/x-icon"
+      href="./images/favicon.svg"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=" "
+    >
```

Comment:
Need to fix code style everywher
```suggestion
    <link
      rel="preconnect"
      href="https://fonts.gstatic.com"
      crossorigin=" "
    >
```